Gemfile

```diff
@@ -59,7 +58,7 @@ gem 'rabl'
 # Once Rails is updated to 5.x we should bump directly to 0.10.x
 gem "active_model_serializers", "0.8.4"
 gem 'oj'
-gem 'deface', '1.0.0'
+gem 'deface'
```

Comment:
I even wonder if we need to specify deface ourselves. It is Spree itself that depends on it on a particular version.